course/models.py

```diff
@@ -9,9 +12,14 @@ def __str__(self):
         return self.name
 
 class Session(models.Model):
```

Comment:
This should be renamed -- this is really a course instance.  A session should be just a time slot of configurable length where a course instance can fit.